projects/plugins/jetpack/_inc/client/security/sso.jsx

```diff
@@ -97,11 +102,12 @@ export const SSO = withModuleSettingsFormHelpers(
 								onChange={ this.handleMatchByEmailToggleChange }
 							>
 								<span className="jp-form-toggle-explanation">
-									{ __( 'Match accounts using email addresses', 'jetpack' ) }
+									{ __( 'Match accounts using email addresses', 'jetpack' ) } (
+									{ this.state.jetpack_sso_match_by_email ? 'yes1' : 'no2' })
```

Comment:
```suggestion
									{ __( 'Match accounts using email addresses', 'jetpack' ) } 
```

It looks like this was just left in for testing?

Reply:
Thank you so much for catching that!